transports/noise/src/protocol/x25519.rs

```diff
@@ -120,6 +120,7 @@ impl Protocol<X25519> for X25519 {
         Ok(PublicKey(X25519(pk)))
     }
 
+    #[allow(irrefutable_let_patterns)]
```

Comment:
This is unfortunate :/

Reply:
Yes, but I don't know how to change this without a massive rewrite :(